proto/p4/config/p4info.proto

```diff
@@ -21,13 +21,11 @@ import "google/protobuf/any.proto";
 package p4.config;
 
 message P4Info {
-  repeated HeaderField header_fields = 1;
```

Comment:
With these two gone, how does a fixed-function target tie the table match-fields to what they semantically mean. For example:
header ipv4_base_t  foo;
reads "foo.ttl"
How (based solely on P4Info) does a target deduce that foo.ttl actually means an IPv4 TTL field?
(I believe we we thinking that we would standardize Header/HeaderField names and keep a separate canonical mapping from those to target-specific "qualifiers" of what these fields stand for)